crypto/README.md

```diff
@@ -90,29 +90,29 @@ Checks whether the given signature is valid.
 
 ### Hash functions
 
-**`sha256Hash = (input: BigInt[]): BigInt`**: a wrapper function over
+**`sha256Hash = (input: bigint[]): bigint`**: a wrapper function over
```

Comment:
Thoughts on having these explanations in the README? Seems like it may become redundant work with TypeDoc.

Reply:
Good point! I'll amend the Readme